index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario= prompt('Ingrese su nombre');
+alert(`Hola ${nombreUsuario}`);
```

Comment:
no me parace que uses interpolacion
```suggestion
alert('Hola '+nombreUsuario);
```